sympy/physics/tests/test_units.py

```diff
@@ -20,6 +20,8 @@ def test_units():
     t = Symbol('t')
     assert integrate(t*m/s, (t, 1*s, 5*s)) == 12*m*s
     assert (t * m/s).integrate((t, 1*s, 5*s)) == 12*m*s
+    
+    abs((year / week).evalf() - 52) < 0.2
```

Comment:
I did evalf to 15 digits precision, then cast it to a Python float for comparison.

(Thank you for your good advice and your patience. I really hope this eventually gets accepted.)